packages/puppeteer-core/src/cdp/Browser.ts

```diff
@@ -345,8 +347,8 @@ export class CdpBrowser extends BrowserBase {
     return this.#connection.url();
   }
 
-  override async newPage(): Promise<Page> {
-    return await this.#defaultContext.newPage();
+  override newPage(): Promise<Page> {
+    return this.#defaultContext.newPage();
```

Comment:
Was required by `eslint`

Reply:
Readd `async` to the function and EsLint will enforce the `await`. Or there is another rule (maybe your IDE adds) that makes this disappear ?

Reply:
During `npm run format`,  the await is automatically added, after which another rule is happened to be broken:
>  514:12  error  Returning an awaited value that is not a promise is not allowed  @typescript-eslint/return-await

Reply:
never mind, looks like it was a problem with local setup which is now gone